go.mod

```diff
@@ -3,53 +3,62 @@ module github.com/GoogleCloudPlatform/spark-on-k8s-operator
-	github.com/Azure/go-autorest v11.7.0+incompatible // indirect
+	cloud.google.com/go v0.38.0
 	github.com/aws/aws-sdk-go v1.29.11
 	github.com/docker/spdystream v0.0.0-20181023171402-6480d4af844c // indirect
 	github.com/elazarl/goproxy v0.0.0-20200421181703-e76ad31c14f6 // indirect
-	github.com/evanphx/json-patch v4.1.0+incompatible
-	github.com/fatih/camelcase v1.0.0 // indirect
+	github.com/evanphx/json-patch v4.2.0+incompatible
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/google/go-cloud v0.1.1
-	github.com/google/gofuzz v1.0.0 // indirect
 	github.com/google/uuid v1.1.1
 	github.com/googleapis/gnostic v0.2.0 // indirect
-	github.com/gophercloud/gophercloud v0.0.0-20190408160324-6c7ac67f8855 // indirect
 	github.com/gregjones/httpcache v0.0.0-20190212212710-3befbb6ad0cc // indirect
-	github.com/hashicorp/golang-lru v0.5.1 // indirect
 	github.com/imdario/mergo v0.3.7 // indirect
-	github.com/json-iterator/go v1.1.6 // indirect
 	github.com/mattn/go-runewidth v0.0.4 // indirect
-	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
-	github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742 // indirect
 	github.com/olekukonko/tablewriter v0.0.2-0.20190409134802-7e037d187b0c
-	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v0.9.3
 	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
 	github.com/robfig/cron v1.2.0
 	github.com/spf13/cobra v0.0.6
 	github.com/stretchr/testify v1.4.0
-	golang.org/x/crypto v0.0.0-20190404164418-38d8ce5564a5 // indirect
 	golang.org/x/net v0.0.0-20200202094626-16171245cfb2
-	golang.org/x/oauth2 v0.0.0-20190402181905-9f3314589c9a // indirect
-	golang.org/x/sync v0.0.0-20190227155943-e225da77a7e6
-	golang.org/x/sys v0.0.0-20190405154228-4b34438f7a67 // indirect
+	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
 	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
-	google.golang.org/api v0.3.2 // indirect
-	google.golang.org/appengine v1.5.0 // indirect
-	gopkg.in/inf.v0 v0.9.1 // indirect
-	k8s.io/api v0.0.0-20190222213804-5cb15d344471
-	k8s.io/apiextensions-apiserver v0.0.0-20190325193600-475668423e9f
-	k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
-	k8s.io/cli-runtime v0.0.0-20190221101700-11047e25a94a // indirect
-	k8s.io/client-go v0.0.0-20190307161346-7621a5ebb88b
-	k8s.io/klog v0.2.0 // indirect
-	k8s.io/kube-openapi v0.0.0-20190401085232-94e1e7b7574c // indirect
-	k8s.io/kubernetes v1.13.5
+	k8s.io/api v0.16.6
+	k8s.io/apiextensions-apiserver v0.0.0
+	k8s.io/apimachinery v0.16.9-beta.0
+	k8s.io/client-go v0.16.9-beta.0
+	k8s.io/kubectl v0.0.0
+	k8s.io/kubernetes v1.16.2
 	k8s.io/utils v0.0.0-20200414100711-2df71ebbae66 // indirect
-	sigs.k8s.io/yaml v1.1.0 // indirect
-	volcano.sh/volcano v0.0.0-20190814060827-e5e4a32196c9
+	volcano.sh/volcano v0.4.0
+)
+
+replace (
+	k8s.io/api => k8s.io/api v0.16.6
```

Comment:
Which k8s version does `v0.16.6` correspond to?

Reply:
1.16.6

Reply:
All the k8s related dependencies are 1.16.6 now